pkg/analysis/passes/legacyplatform/legacyplatform.go

```diff
@@ -47,19 +50,52 @@ func (d *regexDetector) Detect(moduleJs []byte) bool {
 }
 
 var legacyDetectors = []detector{
-	&containsBytesDetector{pattern: []byte("PanelCtrl")},
-	&containsBytesDetector{pattern: []byte("QueryCtrl")},
-	&containsBytesDetector{pattern: []byte("app/plugins/sdk")},
-	&containsBytesDetector{pattern: []byte("angular.isNumber(")},
-	&containsBytesDetector{pattern: []byte("editor.html")},
-	&containsBytesDetector{pattern: []byte("ctrl.annotation")},
 	&containsBytesDetector{pattern: []byte("System.register(")},
 
-	// &regexDetector{regex: regexp.MustCompile(`['"](app/core/.*?)|(app/plugins/.*?)['"]`)},
+	&regexDetector{regex: regexp.MustCompile(`['"](app/core/.*?)|(app/plugins/.*?)['"]`)},
 	&regexDetector{regex: regexp.MustCompile(`['"](app/core/utils/promiseToDigest)|(app/plugins/.*?)|(app/core/core_module)['"]`)},
 	&regexDetector{regex: regexp.MustCompile(`from\s+['"]grafana\/app\/`)},
```

Comment:
I *think* we should remove this `legacyDetectors` altogether and use only GCOM as the only source of truth for angular detections.

In case there are patterns here that are not in GCOM API, then we have removed them from GCOM because they were redundant.